cluster/examples/cockroach/class.yaml

```diff
@@ -1,24 +1,9 @@
-apiVersion: v1
-kind: Namespace
-metadata:
-  name: app-project1-dev
----
-apiVersion: database.crossplane.io/v1alpha1
-kind: PostgreSQLInstanceClass
-metadata:
-  name: postgresql-standard
-  namespace: app-project1-dev
-classRef:
-  kind: CockroachClusterClass
-  apiVersion: database.rook.crossplane.io/v1alpha1
-  name: cockroach-standard
-  namespace: rook-infra-dev
----
 apiVersion: database.rook.crossplane.io/v1alpha1
 kind: CockroachClusterClass
 metadata:
   name: cockroach-standard
-  namespace: rook-infra-dev
+  labels:
+    default: "true"
```

Comment:
Nitpick, but I'd suggest selecting on some other label than `default`, as it could be easily confused with the default annotation, which is `resourceclass.crossplane.io/is-default-class: "true"`

Reply:
@negz Oh yes, I meant to add the full default annotation. Are you fine with using that?

Reply:
SGTM